src/rust/model-plugin-deployer/tests/test_deploy_model.rs

```diff
@@ -0,0 +1,40 @@
+#![cfg(feature = "integration")]
+// ^ Marks the entire file, including helpers, as "only compile for integration tests"
+
+use model_plugin_deployer::client::{
+    DeployModelRequest,
+    ModelPluginDeployerRpcClient,
+};
+
+use tonic::{
+    Code,
+};
+
```

Comment:
maybe exercise the healthcheck endpoints here too?

Reply:
A) Good idea
B) I want to shift over to other more important projects - this is more about getting this project out of PR hell - so I filed a ticket: https://github.com/grapl-security/issue-tracker/issues/760